pycfmodel/model/resources/genericresource.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseModel, Extra, validator
+
+from .resource import Resource
+
+
+class GenericResource(Resource):
+    """This class is used for all resource types that we haven't had time to implement yet"""
+
+    Type: str
+
+    class Config(BaseModel.Config):
+        extra = Extra.allow
+
+    @validator("Type")
+    def check_type(cls, value):
+        return value
```

Comment:
Could we add something here that checks if the type is one of he already supported ones? We could use it to detect an implemented Resource that's failing to match the current model and it's defaulting to GenericResource

Reply:
Something in the lines of:
```
EXISTING_RESOURCE_TYPES = [klass for klass in ResourceModels.__args__]
```